libs/service-portal/settings/access-control/src/screens/Access/Access.tsx

```diff
@@ -1,8 +1,36 @@
 import React from 'react'
+import { useHistory } from 'react-router-dom'
+import { useForm, FormProvider } from 'react-hook-form'
 import { defineMessage } from 'react-intl'
 
-import { Box } from '@island.is/island-ui/core'
+import {
+  Box,
+  Inline,
+  Text,
+  Button,
+  Table as T,
+} from '@island.is/island-ui/core'
 import { IntroHeader } from '@island.is/service-portal/core'
+import { useLocale } from '@island.is/localization'
+
+import { AccessItem } from './components'
+
+const accessGroups = [
+  {
+    id: '1234567890',
+    displayName: 'Mín gögn',
+    description: 'Útskýring',
+    validTo: '',
+    scopes: [
+      {
+        id: '1337',
+        displayName: 'Undiraðgangur',
+        description: 'Útskýring',
+        validTo: '',
+      },
+    ],
+  },
+]
```

Comment:
I assume we'll fetch groups/scopes to list, separately from the delegation which lists which scopes the user has, and how long they're valid.

Reply:
Yeah, I just decided to stop there and work on getting the actual data in.